lib/beaker/dsl/install_utils.rb

```diff
@@ -708,8 +708,8 @@ def install_puppet(opts = {})
             end
           end
 
-          # Certain install paths may not create the config dirs/files needed
-          on host, "mkdir -p #{host['puppetpath']}" unless host[:type] =~ /aio/
+          host.mkdir_p host['puppetpath'] unless (host[:type] =~ /aio/ || host['platform'] =~ /windows/)
```

Comment:
This will no longer execute at all on windows, cygwin or not - is that the desired state?